public/javascripts/payments.js

```diff
@@ -85,8 +87,6 @@
    * of the page to let users pay in just a click (or a tap on mobile).
    */
 
-  // Make sure all data is loaded from the store to compute the order amount.
-  await store.loadProducts();
```

Comment:
this was the duplicate request, but it was in here because it wasn't guaranteed store would load first on init.